### PR TITLE
data: ayush24k

### PR DESCRIPTION
**Pull request by ayush24k**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4734"></a>

### CODE REVIEW

File: data/ayush24k.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/ayushanshuu",
+      "icon": "FaInstagram"
+    },
+      {
+      "name": "Join our WeHelpDev Community",
+      "url": "https://discord.com/invite/VPy2Vhqfp8",
+      "icon": "FaDiscord"
+    },
```
Comment (priyankarpal) [inline data/ayush24k.json:37]: Delete the comma
```suggestion
    }
```

File: data/ayush24k.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Anshu",
+  "DisplayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am Frontend Web Developer"
+  "avatar": "https://github.com/ayush24k.png",
```
Comment (priyankarpal) [inline data/ayush24k.json:6]: Delete the avatar code 
```suggestion

```

File: data/ayush24k.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Ayush Anshu",
+  "DisplayStatsPublic": true,
```
Comment (priyankarpal) [inline data/ayush24k.json:3]: After the current update, you don't need to add this code. Check the [updated doc](https://linkfree.eddiehub.io/docs/quickstart)

```suggestion 
 
```

**Review by priyankarpal — DISMISSED** [review]

**Review by priyankarpal — DISMISSED** [review]
LGTM 🎉

**Review by krshkun — COMMENTED** [review]
Looks good! Gh actions will format the PR ✅

**Review by Hrishi5111998 — APPROVED** [review]

File: data/ayush24k.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Ayush Anshu",
+  "type": "personal",
+  "bio": "I am Frontend Web Developer"
```
Comment (SaraJaoude) [inline data/ayush24k.json:4]:
```suggestion
  "bio": "I am Frontend Web Developer",
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by ayush24k** [comment]
> looks good

Thanks ⚡

**Pull request merged**